ONSdigital/takeon-business-layer
# Feature/spp 1605 add domain group

# Description
Added Domain and ResultsCellNumber columns of Contributor table to a given snapshot

# How to test?
Environment: SPP1515
1. Snapshot can be tested by dropping the following message into the queue 'spp-es-takeon-egestion-spp1515-db-export-input'
{ "snapshot_id": "14e0fb27-d450-44d4-8452-9f6996b00e27",
"surveyperiods": [
{ "survey": "023", "period": "201904" },
{ "survey": "023", "period": "201903" }
] }
2. Verify the success or failure of the snapshot in the output queue 'spp-es-takeon-egestion-spp1515-db-export-output'
3. Verify the snapshot file in the S3 bucket and make sure the Domain and ResultsCellNumber columns of Contributor table exists in the created snapshot file.
S3 Bucket Name: spp-es-takeon-egestion-spp1515-export
4. And also perform the regression testing of Selective Editing by invoking the following lambda 'spp-es-takeon-validation-spp1515-selective-editing' and by creating the following test configuration
{
  "reference": "12000534932",
  "period": "201904",
  "survey": "023"
}

# Verify Check style errors
From the terminal please run
mvn package install
# Verify Unit tests
mvn clean test

# Links
https://collaborate2.ons.gov.uk/jira/browse/SPP-1605
